tracim/views/core_api/session_controller.py

```diff
@@ -99,28 +97,36 @@ def bind(self, configurator: Configurator):
         configurator.add_route(
             'login',
             os.path.join(BASE_API_V2, 'sessions', 'login'),
-            request_method='GET'
+            request_method='POST',
         )
         configurator.add_view(
             self.login,
             route_name='login',
         )
         # Logout
         configurator.add_route(
-            'logout',
+            'post_logout',
```

Comment:
You may write `configurator.add_route('logout', '/sessions/logout'), request_method='POST')` on one line, probably more readable (add # nopep8 at the end if required)

Do not name youre routes with `post_` prefix because it's not necessary for understanding the code and you'll never add two different views with same name but different HTTP method (otherwise this is one unique view)

Reply:
Seems complicated to not use 2 different route_name in pyramid in this case. I do not find a nice way to do this. Thats why there is 'post_logout' and 'get_logout' here.